uefi-test-runner/src/proto/nvme/pass_thru.rs

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use core::time::Duration;
+use uefi::boot;
+use uefi::proto::device_path::text::{AllowShortcuts, DisplayOnly};
+use uefi::proto::device_path::DevicePath;
+use uefi::proto::media::block::BlockIO;
+use uefi::proto::nvme::pass_thru::NvmePassThru;
+use uefi::proto::nvme::{NvmeQueueType, NvmeRequestBuilder};
+
+pub fn test() {
+    info!("Running NVMe PassThru tests");
+
+    assert!(try_find_drive());
+}
+
+fn try_find_drive() -> bool {
```

Comment:
nit: `try_` functions return a `Result` by convention. Maybe `has_nvme_Device` is a better name?